v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -121,6 +132,14 @@ public static PipelineResult run(ProtegrityDataTokenizationOptions options) {
                                     .setErrorRecordsTable(options.getBigQueryTableName() + DEFAULT_DEADLETTER_TABLE_SUFFIX)
                                     .setErrorRecordsTableSchema(SchemaUtils.DEADLETTER_SCHEMA)
                                     .build());
+        } else {
```

Comment:
I guess we should handle this case with `else if` because we can't guarantee that if BQ isn't set then BT is set. If none of the sinks is provided, we should raise an error.

Reply:
Agree, fixed